core/server/worker/src/test/java/alluxio/worker/page/PagedBlockStoreCommitBlockTest.java

```diff
@@ -0,0 +1,274 @@
+import java.nio.ByteBuffer;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class PagedBlockStoreCommitBlockTest {
+  BlockStoreEventListener mListener;
+  UfsManager mUfs;
+  AlluxioConfiguration mConf;
+  CacheManagerOptions mCacheManagerOptions;
+  PagedBlockMetaStore mPageMetaStore;
+  List<PagedBlockStoreDir> mDirs;
+  PagedBlockStore mPagedBlockStore;
+  BlockMasterClientPool mBlockMasterClientPool;
+  BlockMasterClient mMockedBlockMasterClient;
+  AtomicReference<Long> mWorkerId;
+
+  CacheManager mCacheManager;
+
+  private static final int DIR_INDEX = 0;
+
+  private static final Long SESSION_ID = 1L;
+  private static final Long BLOCK_ID = 2L;
+  final int mBlockSize = 64;
+
+  public int mPageSize = 2;
+
+  private static final int OFFSET = 0;
+
+  @Rule
+  public TemporaryFolder mTempFolder = new TemporaryFolder();
+
+  @Before
+  public void setup() throws Exception {
+    List<PageStoreDir> pageStoreDirs;
+    InstancedConfiguration cacheManagerConf = Configuration.copyGlobal();
+
+    Path dirPath = mTempFolder.newFolder().toPath();
+    InstancedConfiguration dirConf = Configuration.modifiableGlobal();
+    dirConf.set(PropertyKey.WORKER_PAGE_STORE_DIRS, ImmutableList.of(dirPath));
+    dirConf.set(PropertyKey.WORKER_PAGE_STORE_SIZES, ImmutableList.of(Constants.MB));
+    dirConf.set(PropertyKey.WORKER_PAGE_STORE_TYPE, PageStoreType.LOCAL);
+    PageStoreDir pageStoreDir =
+            PageStoreDir.createPageStoreDir(
+                    new CacheEvictorOptions().setEvictorClass(FIFOCacheEvictor.class),
+                    PageStoreOptions.createForWorkerPageStore(dirConf).get(DIR_INDEX));
+
+    try  {
+      mUfs = new NoopUfsManager();
+      mConf = Configuration.global();
+      cacheManagerConf.set(PropertyKey.WORKER_PAGE_STORE_PAGE_SIZE, mPageSize);
+      cacheManagerConf.set(PropertyKey.WORKER_PAGE_STORE_DIRS, ImmutableList.of(dirPath));
+
+      // Here mock BlockMasterClientPool and BlockMasterClient since I have no idea
+      // about how to override them.
+      // mockedPool will return a mocked BlockMasterClient when require() is called,
+      // and do nothing when releasing, maybe add some action later on.
+      mBlockMasterClientPool = mock(BlockMasterClientPool.class);
+      mMockedBlockMasterClient = mock(BlockMasterClient.class);
+      when(mBlockMasterClientPool.acquire()).thenReturn(mMockedBlockMasterClient);
+      doNothing().when(mBlockMasterClientPool).release(any());
+      mWorkerId = new AtomicReference<>(-1L);
+      mCacheManagerOptions = CacheManagerOptions.createForWorker(cacheManagerConf);
+      pageStoreDirs = new ArrayList<PageStoreDir>();
+      pageStoreDirs.add(pageStoreDir);
+      mDirs = PagedBlockStoreDir.fromPageStoreDirs(pageStoreDirs);
+    } catch (Exception e) {
+      throw e;
+    }
```

Comment:
throwing immediately after catch is useless

Reply:
2333